packages/unit-testing/stubs/dataset/DatasetInstance.js

```diff
@@ -1,4 +1,4 @@
-define(['./Condition', './Column'], function (Condition, Column) {
```

Comment:
Missing Expression parameter in function.

Reply:
Taking a closer look, I think we should also "inject" query.ResultSet and query.PagedData.